File: src/main/java/com/msg/gauth/domain/auth/presentation/dto/request/SigninRequestDto.java
package com.msg.gauth.domain.auth.presentation.dto.request;

import lombok.Getter;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Pattern;

@Getter
public class SigninRequestDto {
    @NotBlank
    @Pattern(regexp = "^[a-zA-Z0-9]+@gsm.hs.kr$")
    String email;

    @NotBlank
    String password;
}

File: src/main/kotlin/com/msg/gauth/domain/auth/presentation/dto/request/SigninRequestDto.kt
package com.msg.gauth.domain.auth.presentation.dto.request

import javax.validation.constraints.NotBlank
import javax.validation.constraints.Pattern

data class SigninRequestDto(
    @field:NotBlank
    @field:Pattern(regexp = "^[a-zA-Z0-9]+@gsm.hs.kr$")
    val email: String,

    @field:NotBlank
    val password: String
)